lib/gui/components/update-notifier/update-notifier.js

```diff
@@ -21,24 +21,42 @@
  */
 
 const angular = require('angular');
+const _ = require('lodash');
+const release = require('../../../../lib/shared/release');
+const packageJSON = require('../../../../package.json');
+
 const MODULE_NAME = 'Etcher.Components.UpdateNotifier';
 const UpdateNotifier = angular.module(MODULE_NAME, [
   require('../modal/modal'),
   require('../../models/settings'),
-  require('../../utils/manifest-bind/manifest-bind'),
   require('../../os/open-external/open-external'),
   require('../../modules/analytics')
 ]);
 
 /**
- * @summary The number of days the update notifier can be put to sleep
+ * @summary The number of days the update notifier can be put to sleep on production builds
  * @constant
  * @private
  * @type {Number}
  */
-const UPDATE_NOTIFIER_SLEEP_DAYS = 7;
+const UPDATE_NOTIFIER_SLEEP_DAYS_PRODUCTION = 7;
+
+/**
+ * @summary The number of days the update notifier can be put to sleep on non-production builds
+ * @constant
+ * @private
+ * @type {Number}
+ */
+const UPDATE_NOTIFIER_SLEEP_DAYS_OTHER = 0;
+
+UpdateNotifier.constant('UPDATE_NOTIFIER_SLEEP_DAYS', _.attempt((releaseType) => {
+  if (releaseType === release.RELEASE_TYPE.PRODUCTION) {
+    return UPDATE_NOTIFIER_SLEEP_DAYS_PRODUCTION;
+  }
+
+  return UPDATE_NOTIFIER_SLEEP_DAYS_OTHER;
+}, release.getReleaseType(packageJSON.version)));
```

Comment:
I'm *almost* sure that snapshot builds and production builds won't share the same settings, but let me quickly validate that.